tests/redirects/map_globalconf.py

```diff
@@ -780,7 +780,7 @@
         url_test("/de/impressum/", "/de/about/legal/impressum/"),
         # bug 1248393
         url_test("/de/about/legal/impressum/", status_code=requests.codes.ok),
-        url_test("/{en-US,fr,ja}/about/legal/impressum/", "/de/about/legal/impressum/", status_code=requests.codes.found),
+        url_test("/{en-US,fr,ja}/about/legal/impressum/", status_code=requests.codes.not_found),
```

Comment:
I know that the German Impressum page is an important one that only exists in `de`. I can't currently run the code to check, but my assumption here is that the 404 page would have a link to `/de/about/legal/impressum/` yeah?

Reply:
Yep! 👍